zkevm-circuits/src/state_circuit/test.rs

```diff
@@ -98,6 +106,7 @@ fn test_state_circuit_ok(
     let prover = MockProver::<Fr>::run(19, &circuit, power_of_randomness).unwrap();
     let verify_result = prover.verify();
     assert_eq!(verify_result, Ok(()));
+    // prover.assert_satisfied();
```

Comment:
```suggestion
```

Reply:
Resolved in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1046/commits/f59933089f977ec564f751c7051a824c0d5dd041